Controllers/AdminController.cs

```diff
@@ -31,9 +33,9 @@ public IActionResult DeleteBook(long id)
         [HttpGet]
         public IActionResult Index(long id)
         {
-            //TODO: Index will return the list of books (with Author), ordered by SKU
+            //TODO DONE: Index will return the list of books (with Author), ordered by SKU
             //or a single book (with Author), if passed an id > 0
-            IEnumerable<Book> books = _db.Books.Include(b => b.Author);
+            IEnumerable<Book> books = _db.Books.Include(b => b.Author).OrderBy(b => b.SKU);
```

Comment:
This works well to return the entire set of books. 
Also add the logic to return just a single book if the `id > 0`